sloppy-deployment/entrypoint.sh

```diff
@@ -9,6 +11,12 @@ gomplate -f "/work/${SLOPPY_TEMPLATE_FILE}" \
 
 if [[ -n "$DRY_RUN" ]]; then
   cat "${SLOPPY_OUTPUT_FILE}"
+  if [[ $safe_yaml ]] ; then
```

Comment:
Please be DRY: should be a bash function.

Reply:
done